tests/unit/utils/utils_test.py

```diff
@@ -139,10 +139,12 @@ def test_safe_rm(self, os_remove_mock):
 
     @skipIf(os.path.exists('/tmp/no_way_this_is_a_file_nope.sh'), 'Test file exists! Skipping safe_rm_exceptions test!')
     def test_safe_rm_exceptions(self):
+        error = False
         try:
             utils.safe_rm('/tmp/no_way_this_is_a_file_nope.sh')
         except (IOError, OSError):
-            self.assertTrue(False, "utils.safe_rm raised exception when it should not have")
```

Comment:
Perhaps http://stackoverflow.com/questions/4319825/python-unittest-opposite-of-assertraises?answertab=active#tab-top ?